### PR TITLE
[14.0][MIG] account_financial_report: Migration to 14.0

### PR DESCRIPTION
**Pull request by tupaq**
dependencies:

- [x]  date_range

- [x] report_xlsx

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@tupaq Consider adding https://github.com/ForgeFlow/reporting-engine/commit/b6fbc66b5f3c548e7b13517e77f1140ecafb15f7 to fix the error in travis due to dependencies. When report_xlsx adjust the oca dependency as indicated in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca-dependencies-txt

**Comment by tupaq** [comment]
> @tupaq Consider adding [ForgeFlow/reporting-engine@b6fbc66](https://github.com/ForgeFlow/reporting-engine/commit/b6fbc66b5f3c548e7b13517e77f1140ecafb15f7) to fix the error in travis due to dependencies. When report_xlsx adjust the oca dependency as indicated in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca-dependencies-txt
@JordiBForgeFlow it's added and it doesn't work

File: account_financial_report/report/trial_balance_xlsx.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+                    )
+                self.write_account_footer(
+                    accounts_data[account_id],
+                    accounts_data[account_id]["code"]
+                    + "- "
+                    + accounts_data[account_id]["name"],
+                )
+
+                # Line break
+                self.row_pos += 2
```
Comment (coleste) [inline account_financial_report/report/trial_balance_xlsx.py]: Why no `self.set_row_pos()` here?

File: account_financial_report/report/aged_partner_balance_xlsx.py [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+                    # Display account move lines
+                    for line in partner["move_lines"]:
+                        self.write_line_from_dict(line)
+
+                    # Display ending balance line for partner
+                    self.write_ending_balance_from_dict(partner)
+
+                    # Line break
+                    self.row_pos += 1
```
Comment (coleste) [inline account_financial_report/report/aged_partner_balance_xlsx.py]: Why no `self.set_row_pos()` here?

**Comment by pedrobaeza** [comment]
Travis and runbot are red. Can you please check?

File: account_financial_report/report/trial_balance_xlsx.py (hunk 1 above)
Comment (tupaq) [reply account_financial_report/report/trial_balance_xlsx.py]: is fixed

File: account_financial_report/report/aged_partner_balance_xlsx.py (hunk 2 above)
Comment (tupaq) [reply account_financial_report/report/aged_partner_balance_xlsx.py]: is fixed

**Comment by coleste** [comment]
And I don't know if server-ux should be in oca-dependencies-txt, since we depend on date_range.

**Comment by tupaq** [comment]
> And I don't know if server-ux should be in oca-dependencies-txt, since we depend on date_range.

you are right. Patched

**Comment by ventor-dev** [comment]
@tupaq , any news on this migration?

**Pull request closed without merging**